ng-dev/pr/config/index.ts

```diff
@@ -91,8 +92,29 @@ export function assertValidPullRequestConfig<T extends NgDevConfig>(
   }
 }
 
-/** Label for pull requests containing a breaking change. */
-export const breakingChangeLabel = 'flag: breaking change';
+interface Label {
+  /* The label string. */
+  label: string;
+  /** A matching function, if the label is automatically applied by our github action, otherwise false. */
+  commitCheck: ((c: Commit) => boolean) | false;
+}
 
-/** Label for pull requests containing a deprecation. */
-export const deprecationLabel = 'flag: deprecation';
+/** Set of labels which are known to tooling, and in some cases are managed by tooling. */
+export const ToolingPullRequestLabels = {
```

Comment:
Nice! I love how flexible this is.

Do you think we can move this somewhere else outside of `/config/index` because that is meant to be always for the ng-dev configuration part, like e.g. `release/config`. How about `config/constants.ts`?

Reply:
What do you think of `ng-dev/pr/common/labels.ts`?